packages/config/config/devices/0x018c/pm-b400zw-n.json

```diff
@@ -0,0 +1,73 @@
+{
+	"manufacturer": "Dawon DNS",
+	"manufacturerId": "0x018c",
+	"label": "PM-B400ZW-N",
+	"description": "Power Manager",
+	"devices": [
+		{
+			"productType": "0x0042",
+			"productId": "0x0006",
+			"zwaveAllianceId": 1870
+		},
+		{
+			"productType": "0x0042",
+			"productId": "0x0005"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Group 1",
+			"maxNodes": 5,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
```

Comment:
> It appears from the manual that once a value over 0 is added, the device enables the 4th byte (0x01)

Assuming the manual is consistent and the Enable/Disable byte is the 4th (LSB), this needs two partial parameters, one with mask `0xffffff00` for the power value and `0xff` for the enable/disable.